tools/dds/dds-server/lrs-device-controller.cpp

```diff
@@ -182,7 +182,16 @@ void lrs_device_controller::start_streaming( const json & msg )
     _sensors[sensor_index].start( [&]( rs2::frame f ) {
         _dds_device_server->publish_image( f.get_profile().stream_name(), static_cast< const uint8_t * >( f.get_data() ), f.get_data_size() );
     } );
-    std::cout << realdds_streams_to_start[0].first << " stream started" << std::endl;
+    if( realdds_streams_to_start.size() == 1 )
```

Comment:
I see this comment above:
`TODO - currently assumes all streams are from one sensor. Add support for multiple sensors`
Is it still valid?

Reply:
Yes, as a future ability for the tool.
Currently, as we use LibRS at both sides, functionality is per sensor, but generally you can use streams of various sensors in one call